scripts/CMakeLists.txt

```diff
@@ -102,53 +102,6 @@ if (UPDATE_DEPS)
     endif()
 endif()
 
-# Configure compiler build options common to all targets
-function(lunarg_target_compiler_configurations TARGET_NAME WERROR_OPTION)
-    if (NOT ${WERROR_OPTION})
-        return()
-    endif()
-
-    if (MSVC)
-        target_compile_options(${TARGET_NAME} INTERFACE /WX)
-        target_link_options(${TARGET_NAME} INTERFACE /WX)
-    else()
-        target_compile_options(${TARGET_NAME} INTERFACE -Werror)
-    endif()
-
-    if(${CMAKE_CXX_COMPILER_ID} MATCHES "(GNU|Clang)")
-        target_compile_options(${TARGET_NAME} INTERFACE
-            -Wpedantic
-            -Wunreachable-code
-            -Wunused-function
-            -Wall
-            -Wextra
-            -Wpointer-arith
-        )
-        if(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
-            target_compile_options(${TARGET_NAME} INTERFACE
-                -Wunreachable-code-return
-                -Wconversion
-                -Wimplicit-fallthrough
-                -Wstring-conversion
-            )
-        endif()
-    elseif(MSVC)
-        target_compile_options(${TARGET_NAME} INTERFACE
-            /W4
-            /we5038 # Enable warning about MIL ordering in constructors
-        )
-
-        # Enforce stricter ISO C++
-        target_compile_options(${TARGET_NAME} INTERFACE /permissive-)
-
-        target_compile_options(${TARGET_NAME} INTERFACE $<$<BOOL:${MSVC_IDE}>:/MP>) # Speed up Visual Studio builds
-
-        # Allow usage of unsafe CRT functions and minimize what Windows.h leaks
-        target_compile_definitions(${TARGET_NAME} INTERFACE _CRT_SECURE_NO_WARNINGS NOMINMAX WIN32_LEAN_AND_MEAN)
```

Comment:
Note: I didn't want to export `_CRT_SECURE_NO_WARNINGS` since that should be a decision left to the authors of each library.